src/components/Common/UI/Breadcrumbs/BreadcrumbsTypes.ts

```diff
@@ -0,0 +1,28 @@
+import type { ReactNode } from 'react'
+
+export interface Breadcrumb {
+  id: string
+  label: ReactNode
+}
+export interface BreadcrumbsProps {
+  /**
+   * Array of breadcrumbs
+   */
+  breadcrumbs: Breadcrumb[]
+  /**
+   * Current breadcrumb id
+   */
+  currentBreadcrumbId?: string
+  /**
+   * Accessibility label for the breadcrumbs
+   */
+  ariaLabel?: string
```

Comment:
Nit: i believe we type this as `'aria-label'` in other instances since we're just using the html props